### PR TITLE
Added package deletion workflow and usernames with package delete access

### PR DESCRIPTION
**Pull request by dilanSachi**

### CODE REVIEW

File: .github/workflows/package_delete_usernames.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      "username": "anupama-pathirage"
+    },
+    {
+      "username": "manuranga"
+    },
+    {
+      "username": "niveathika"
+    }
+  ]
+}
```
Comment (keizer619) [inline .github/workflows/package_delete_usernames.json:19]: @dilanSachi Lets include a new line in the end

File: .github/workflows/delete-package.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      - name: Set version ID
+        id: setid
+        if: steps.setUname.outputs.foundUname != ''
+        run: printf "::set-output name=versionId::%s" $(cat /tmp/filtered-id.json)
+      - name: Delete selected package
+        uses: actions/delete-package-versions@v1.0.3
+        if: steps.setUname.outputs.foundUname != '' && steps.setid.outputs.versionId != ''
+        with:
+          package-version-ids: ${{ steps.setid.outputs.versionId }}
+          token: ${{ secrets.PACKAGE_DELETE_PAT }}
```
Comment (keizer619) [inline .github/workflows/delete-package.yml:49]: @dilanSachi Lets include a new line in the end

File: .github/workflows/package_delete_usernames.json (hunk 1 above)
Comment (dilanSachi) [reply .github/workflows/package_delete_usernames.json]: @keizer619 done

File: .github/workflows/delete-package.yml (hunk 2 above)
Comment (dilanSachi) [reply .github/workflows/delete-package.yml]: @keizer619 done

**Review by keizer619 — APPROVED** [review]

**Comment by keizer619** [comment]
@dilanSachi 
`Added package deletion workflow and usernames with package delete access`
As a practice we keep commit message verb in present tense. We follow following approach
https://chris.beams.io/posts/git-commit/

**Pull request merged**